R/regressionlinear.R

```diff
@@ -1233,8 +1233,12 @@ RegressionLinearInternal <- function(jaspResults, dataset = NULL, options) {
 
       }
 
+      # get translation for (Intercept) 
+      name = names[i]
+      if (identical(name, "(Intercept)")) { name = gettext(name) }
```

Comment:
```suggestion
      name <- names[i]
      if (identical(name, "(Intercept)")) 
        name <- gettext("(Intercept)")
```
I would explicitly write `gettext("(Intercept)")`. Otherwise, you assume that there is a call to `gettext("(Intercept)")` elsewhere in the code. Without any calls to `gettext("(Intercept)")` the string `(Intercept)` will not be detected and cannot be translated.